_assignments/mp1.md

```diff
@@ -0,0 +1,444 @@
+Right above the "Secondary Maps" subheading is the "Tiling" option, which has an option for `x` and `y`.
+Tiling causes a material to repeat itself on the same object, rather than covering the whole thing.
+So, changing tiling `x` to 2, means that the material will repeat once, and show up twice, in the x direction on the wall.
+Play with the tiling until you like the look of it.
+Below is an example of non tiled and tiled walls side by side.
+
+![Tiling](/img/assignments/mp1/image1.png)
+
+In the inspector tab, right below the albedo option, there are metallic and smoothness sliders.
+Play around with these, and see how they affect the material.
+The metallic slider adjusts the reflections, and smoothness helps to enhance or subdue the normal map.
+Modify the metallicity and smoothness of the two wall materials so that they are clearly visually different.
+
+Finally, create a material, that has no albedo or normal map.
+Next to the albedo option is a small color box.
+Since this material has no albedo, the material will be this flat reflection color.
+Try and see what happens when you change the color of a material with an albedo.
+Apply this flat color onto a third wall in the room.
+
+## Scripting
+
+<!-- Some of this should eventually go in Developer Resources. -->
+
+Unity scripts use the [C# language](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/).
+If you are unfamiliar with programming, you can check out this [C# tutorial](https://www.tutorialspoint.com/csharp/).
+You’ll only need the basics of objects, classes, and variables for now.
+
+We recommend reading the Unity [scripts manual](https://docs.unity3d.com/Manual/CreatingAndUsingScripts.html) and the [script object API](https://docs.unity3d.com/ScriptReference/MonoBehaviour.html).
+
+Unity's [Scripting API Reference](https://docs.unity3d.com/ScriptReference/) is a useful source of information for when you are scripting new objects.
+For the first script, we will detail the functions that we recommend,
+but for the others, we expect that you will refer to the API reference if needed.
+
+### Side Note: Input in VR
+
+Keyboards do not work well in VR, since the VR environment means that you cannot see the keyboard, and all of the keys largely feel the same.
+Controllers, with their contours and designated button shapes, are much easier to use blind.
+Refer to the following pages for more information, which you will need when you write the scripts below:
+
+- [Unity Input](https://docs.unity3d.com/Manual/ConventionalGameInput.html)
+- [Unity wiki page on Xbox controllers](http://wiki.unity3d.com/index.php?title=Xbox360Controller)
+- OVR Input: [general explanation](https://developer.oculus.com/documentation/unity/latest/concepts/unity-ovrinput),
+[API reference](https://developer.oculus.com/reference/unity/1.40/class_o_v_r_input/)
+
+Your control scheme must use **controller**, not keyboard, input.

```

Comment:
Should we push that student should use the Oculus Touch?

Reply:
From what I remember, every computer had an XBox controller, but not every computer has touch controllers. Has this been resolved?